### PR TITLE
Added initial layout. Next step added lammps workflow example

### PR DESCRIPTION
**Pull request by AcerP-py**
This is just a draft. Several of the containers still need to be built (all need to be tested). And I plan to write an example of how to build in the container off-site and then run it on frontier by binding in the libs using lammps.

### CODE REVIEW

File: software/containers_on_frontier.rst [hunk 1]
```diff
@@ -122,12 +122,68 @@ This example will look at building a container image for testing collective comm
...
+
+To make the use of apptainer easier, OLCF provides some modules that automatically bind in the needed libraries to run
+apptainer with the host mpi and rocm. To accsess these modules load ``olcf-container-tools``. You should then see two
+modules ``apptainer-enable-mpi`` and ``apptainer-enable-gpu``.
+
+Example Workflow
+^^^^^^^^^^^^^^^^
+To see how one might use these containers and modules we have an example of building and running lammps. You can
+find examples for cpu and gpu lammps runs `here <https://github.com/olcf/olcf_containers_examples/tree/main/frontier/containers_on_frontier_docs/apptainer_wrappers_lammps>`_.
+Clone the git repo on any ``x86_64`` machine, navigate to the correct folder and run:
```
Comment (secondspass) [inline software/containers_on_frontier.rst:177]: User can build the container with no problem on Frontier as well, so indicate that adjust the wording that says "after the image is built transfer it to Frontier". So you could write something like:

```
Clone the git repo onto Frontier (or any ``x86_64`` machine), navigate to the correct folder and run:
.. code-block::
    apptainer build lammps.sif lammps.def
After the image is built transfer it to Frontier if its on another machine, and run it by submitting the ``submit.slurm`` batch script that accompanies it.

File: software/containers_on_frontier.rst [hunk 2]
```diff
@@ -122,12 +122,68 @@ This example will look at building a container image for testing collective comm
...
 
+OLCF Base Images & Apptainer Modules
+-------------------------------------
+To assist the container workflow on Frontier, OLCF provides some base container images and apptainer modules to
+simplify the process. The following sections document them and provide an `Example Workflow`_.
 
+Base Images
+^^^^^^^^^^^
+
+Due to copyright issues, OLCF is not able to provide containers with the Cray Programming Environment (CPE) installed
```
Comment (secondspass) [inline software/containers_on_frontier.rst:133]: I would reword this to something like: "Due to licensing, OLCF is currently not able to provide containers with the Cray Programming Environment (CPE) installed in them"

**Review by secondspass — CHANGES_REQUESTED** [review]
Thanks for working on this! I've made a couple of comments so please take a look

File: software/containers_on_frontier.rst (hunk 1 above)
Comment (AcerP-py) [reply software/containers_on_frontier.rst]: Fixed in [b1069b3](https://github.com/olcf/olcf-user-docs/pull/870/commits/b1069b3a71210e8864cc6eeb20f70d7e5c8e53e7)

File: software/containers_on_frontier.rst (hunk 2 above)
Comment (AcerP-py) [reply software/containers_on_frontier.rst]: Fixed in [b1069b3](https://github.com/olcf/olcf-user-docs/pull/870/commits/b1069b3a71210e8864cc6eeb20f70d7e5c8e53e7)

**Review by secondspass — APPROVED** [review]
